cmd/controller-manager/app/controller-manager.go

```diff
@@ -190,10 +193,13 @@ func getFederationConfig(opts *options.Options) *corev1a1.FederationConfig {
 		}
 
 		err := client.Get(context.Background(), fedConfig, namespace, name)
-		if err != nil {
+		if err != nil && apierrors.IsNotFound(err) {
```

Comment:
`err != nil` appears unnecessary here.

Reply:
OK.

I use it because other places use it in similar way. 
However, I checked related code, and test locally. if `err` is `nil`, `apierrors.IsNotFound(err)` returns false.